amical/tests/test_extraction.py

```diff
@@ -0,0 +1,23 @@
+import munch
+from munch import munchify as dict2class
```

Comment:
I believe this aliasing was mostly introduced as a shortcut in amical, but in the context of a test, I don't think it's wise to replicate this kind of obfuscation actually
```suggestion
```

Reply:
I added both suggested changes related to this in a common commit. Thanks!